src/main/kotlin/no/nav/familie/ef/iverksett/startIverksett/infrastruktur/IverksettController.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.familie.ef.iverksett.startIverksett.infrastruktur
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import no.nav.familie.ef.iverksett.infrastruktur.json.IverksettJson
+import no.nav.familie.ef.iverksett.lagreIverksett.tjeneste.LagreIverksettService
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/api"])
```

Comment:
Jeg syns det er fint hvis requestmappingen på klassenivå er basepath for controllern, dvs `/api/iverksett` :)

Reply:
ja, legger iverksett på klassenivå